kafka-client/src/main/java/dev/responsive/db/CassandraKeyValueSchema.java

```diff
@@ -302,22 +365,4 @@ protected static KeyValue<Bytes, byte[]> rows(final Row row) {
     );
   }
 
-  @Override
-  public int compare(final Bytes o1, final Bytes o2) {
-    return Objects.compare(o1, o2, Bytes::compareTo);
-  }
-
-  // Visible for Testing
-  public static class KeyValueMetadataKeys implements MetadataKeys {
```

Comment:
this was just a confusing pattern, I removed this so each schema is self contained instead of having `MetadataStatements`